### PR TITLE
Feat: add docker image for projectScanner

### PR DESCRIPTION
**Pull request by Tailp**
Hi :) @monperrus , @gluckzhang , this is the pure java docker image for the projectScanner.

### CODE REVIEW

**Comment by monperrus** [comment]
Cool!

We can further simplify this:

* move the commands of `build_scanner.sh` and `configure_git.sh` directly in `Dockerfile`. 
* Remove `scanner_launcher.sh` and call the Java process directly as ENTRYPOINT.

**Comment by gluckzhang** [comment]
Nice work and agree with Martin! Thanks @Tailp :)

File: repairnator/docker-images/projectscanner-dockerimage/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+RUN apt-get install cloc -y
+RUN /root/configure_git.sh
+RUN /root/build_scanner.sh
+
+# Possible values: NOOP (does nothing, usual project scanner), KUBERNETES (receive/send to ActiveMQ)
+ENV PIPELINE_MODE=NOOP
+
+# Github information is provided if the repaired builds should be submitted on some repo
+# optional for something can be empty
+# Number of hours to inspect to get last builds (e.g. 4 will mean 4 hours in the past)
```
Comment (gluckzhang) [inline repairnator/docker-images/projectscanner-dockerimage/Dockerfile]: This comment line seems to explain `ENV SCANNER_NB_HOURS=1`, which could be put just before that line of code.

**Comment by Tailp** [comment]
@gluckzhang @monperrus hi :) , thank you very much for the tips. It's resolved. Waiting for new comments.

**Comment by gluckzhang** [comment]
Thanks, it looks good to me :)

**Comment by monperrus** [comment]
Thanks!

**Comment by iamKunalGupta** [comment]
@Tailp Could you add a README on how to run this Dockerfile, I get the following errors:
```java
18:44:09.071 [main] ERROR f.i.s.r.s.e.j.JSONFileSerializerEngine - Error while creating file writer for: /root/build_list.txt/scanner.json
java.io.FileNotFoundException: /root/build_list.txt/scanner.json (Not a directory)
```

I just want to run repairnator inside docker.

**Comment by Tailp** [comment]
Hi @iamKunalGupta :) , this is just the project scanner . Currently, It is mainly meant for using together with ActiveMQ on Kubernetes. If you just want to fetch failed build ids to later repair with repairnator-pipeline it can be done  just fine with the script inside repairnator/script folder . 

If you absolutely must run docker then here are some commands for that
```
docker run --net=host webcenter/activemq
docker run --net=host -e ACTIVEMQ_URL=tcp://localhost:61616 -e PIPELINEMODE=KUBERNETES SCANNER_IMAGENAME
```
Then you can feed it the publisher python script inside `kubernetes-support` folder with a slug 
```
python -d /queue/scanner Tailp/travisplay
```
It will start scanning and if there are some failed Java builds found it will submit to a queue called `pipeline` if you visit activeMQ server web with `localhost:8161`.

**Pull request merged**